scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -39,156 +39,96 @@ def __init__(self, created_at, updated_at, description, host_url, name, type):
-                f"{bcolors.FAIL} {bcolors.BOLD}FAILURE: {bcolors.ENDC}{bcolors.FAIL}Migration script execution failed. Please contact the support team{bcolors.ENDC}")
-            raise Exception("FAILURE: Migration script execution failed. Please contact the support team.")
-        else:
-            logging.info(f"{bcolors.OKGREEN}{bcolors.BOLD}Successfully completed the migration.{bcolors.ENDC}")
-            print(f"{bcolors.OKGREEN}{bcolors.BOLD}Successfully completed the migration.{bcolors.ENDC}")
-            commit_transactions()
+        logging.info('Updating databases from v3.12.x to v4.0.x')
+
+        logging.info("Dropping audit db table delivery_insights_chart_counts")
+        print("Dropping audit db table delivery_insights_chart_counts")
+        dropDeliveryInsights()
+        logging.info("Dropping audit db table area_chart_counts")
+        print("Dropping audit db table area_chart_counts")
+        dropAreaCharts()
+
+        logging.info("Creating audit db table delivery_insights_chart_counts")
+        print("Creating audit db table delivery_insights_chart_counts")
+        createDeliveryInsights()
+
+        logging.info("Creating audit db table area_chart_counts")
+        print("Creating audit db table area_chart_counts")
+        createAreaCharts()
+        audit_conn.commit()
+            
+
+        logging.info("Altering platform db table app_environment")
+        print("Altering platform db table app_environment")
+        alterAppEnvironmentTable()
+
+        logging.info("Updating spinnaker environments details in sapor db")
+        print("Updating spinnaker environments details in sapor db")
+        getEnvironmentData()
+
+        logging.info("Updating spinnaker environments Id details in app_environment table of platform db")
+        print("Updating spinnaker environments Id details in app_environment table of platform db")
+        environmentUpdate()
+            
+
+        logging.info("Altering platform db table service_gate")
+        print("Altering platform db table service_gate")
+        alterServiceGateTable()
+
+        logging.info("Updating ref_id in service_gate table")
+        print("Updating ref_id in service_gate table")
+        updateRefId()
+
+        logging.info("Altering platform db table service_deployments_current")
+        print("Altering platform db table service_deployments_current")
+        add_columns_in_service_deployment_current()
+
+        logging.info("Altering autopilot db table userlogfeedback and loganalysis")
+        print("Altering autopilot db table userlogfeedback and loganalysis")
+        update_autopilot_constraints()
+
+        logging.info("Updating service_deployments_current table sync column data in platform db")
+        print("Updating service_deployments_current table sync column data in platform db")
+        update_sync_status()
+
+        logging.info("Logging in to ISD")
+        print("Logging in to ISD")
+        #cookie = login_to_isd()
```

Comment:
if you comment / remove the login_to_isd from here, please comment / remove the log message as well.